src/EA.Weee.RequestHandlers.Tests.Unit/Admin/Reports/GetAatfAeReturnDataCsvHandlerTests.cs

```diff
@@ -0,0 +1,130 @@
+﻿namespace EA.Weee.RequestHandlers.Tests.Unit.Admin.Reports
+{
+    using System;
+    using System.Security;
+    using System.Threading.Tasks;
+    using Core.Admin;
+    using Core.Shared;
+    using DataAccess;
+    using EA.Weee.Core.AatfReturn;
+    using FakeItEasy;
+    using RequestHandlers.Admin.Reports;
+    using Requests.Admin.Reports;
+    using Weee.Tests.Core;
+    using Xunit;
+
+    public class GetAatfAeReturnDataCsvHandlerTests
```

Comment:
With these tests, rather than just checking for non empty file result is it possible to actually test the file content is correct.
For example fake the call to SpgAatfAeReturnDataCsvData to return items, then the file content should match the items returns

Reply:
Added